packages/interface/interface.json

```diff
@@ -66,11 +66,6 @@
           "params": "HIG.abstract.EventObject.params",
           "returns": "HIG.abstract.EventObject.returns"
         },
-        "onScroll": {
```

Comment:
This scroll handler is specific to notifications, so it makes sense to remove the handler from general Flyouts.